### ISSUE TITLE
[FEATURE] Update to 1.20

### ISSUE THREAD

**Issue by wyan** [DarkKronicle/AdvancedChatCore#68]
**Describe the solution you'd like**
It'd be lovely if we could have a build for 1.20

**Comment by RimonsVirginity** [comment]
Bumping this!

**Comment by Colt44Magnum** [comment]
bumping again

**Comment by gurrrrrrett3** [comment]
Dark is pretty busy with irl stuff at the moment, however we have some third party forks provided by @medisant 

[Link Here](https://github.com/medisant/AdvancedChatCore/releases/tag/v1.5.10-1.20.2)

**Comment by Colt44Magnum** [comment]
Yeah checked it out! No problem on the irl stuff, it comes before any mod making imo. Actually didn't know this was on 1.19.4 cause I was looking on Curseforge at first. I've been liking the mod so far ~~though I was originally looking for a mod to seperate chat into another program window so I could use a second screen~~

**Comment by gurrrrrrett3** [comment]
I saw someone in our discord that made something for that forever ago I'll see if I can find it in a bit

**Comment by Zoro-6191** [comment]
> Dark is pretty busy with irl stuff at the moment, however we have some third party forks provided by @medisant
> 
> [Link Here](https://github.com/medisant/AdvancedChatCore/releases/tag/v1.5.10-1.20.2)

Error 404 on the files :(
